config/vufind/searches.ini

```diff
@@ -154,6 +154,15 @@ retain_filters_by_default = true
 ; JavaScript is not supported by the browser.
 load_results_with_js = true
 
+; This setting can be used to configure pagination control on top of results.
+; Possible options are:
+; empty string or false  No top pagination (default when load_results_with_js is
```

Comment:
I like the top paginator, though I'd definitely want to hear more input from people more invested in UI design than me. 

However, I don't think the comments about how it behaves match the code -- I don't get different defaults when I turn off AJAX.

Reply:
Default behavior fixed.